forc-pkg/src/manifest.rs

```diff
@@ -833,6 +834,34 @@ impl WorkspaceManifest {
                     member_path
                 );
             }
+            let member_manifest_file = PackageManifestFile::from_file(member_path.clone())?;
+            let pkg_name = member_manifest_file.manifest.project.name;
+            if let Some(paths) = pkg_name_to_paths.get_mut(&pkg_name) {
+                paths.push(member_path);
+            } else {
+                pkg_name_to_paths.insert(pkg_name, vec![member_path]);
+            }
+        }
+
+        // Check for duplicate pkg name entries in member manifests of this workspace.
+        let duplciate_pkg_lines = pkg_name_to_paths
+            .iter()
+            .filter(|(_, paths)| paths.len() > 1)
+            .map(|(pkg_name, _)| pkg_name)
+            .map(|pkg_name| {
+                let duplicate_paths = pkg_name_to_paths
+                    .get(pkg_name)
+                    .expect("missing duplicate paths");
+                format!("{}: {:?}", pkg_name, duplicate_paths)
```

Comment:
```suggestion
                format!("{}: {:#?}", pkg_name, duplicate_paths)
```
^ I _think_ this should print each path on a new line to improve readability a little, but I could be wrong.

Reply:
Just tested with and without this change and pretty print is easily 10x better, thanks!